### DISCUSSION TITLE
Measure WPM and other metrics on certain segment of the race

### DISCUSSION THREAD

**Discussion by yistarostin** [monkeytypegame/monkeytype#1892] [Ideas] [answered]
Hi. Let's you were doing a 30-seconds race and first 10 seconds very perfect, but then you mistyped several words and the whole race got wasted. In this case you might want to get statistics about first 10 seconds of the race, so, I think, that would be a very useful functionality. I shouldn't be very hard to implement, just need to call plot drawing and other metrics calculation for a part of the data. Also we might add 2 buttons/sliders to the post-race screen for choosing the segment of the race.

**Comment by unrealapex** [comment, marked as answer]
Wouldn't it be possible to use the `burst heatmap` feature? You can view it by clicking on the `Toggle words history` button(align left, looks like a block of words) then clicking `toggle burst heatmap` button(fire alt, looks like a flame). See screenshot below:

![burst heatmap](https://user-images.githubusercontent.com/79121360/134281274-8de0b1c1-1e83-4f73-8df6-b56ce5f74571.png)

Alternately, why not look at the post test graph?:

![image](https://user-images.githubusercontent.com/79121360/134281548-20e427fe-9e29-4959-9440-cf7fed1bac61.png)

Did I answer your question?
